scripts/superdesk/upload/styles/upload.less

```diff
@@ -485,13 +486,24 @@ media uploader
 
 					input[type="text"] {
 						border: 0 !important;
-						height: 34px !important;
+						height: 26px !important;
```

Comment:
Yes I know, but it was set before like this to overwrite common input style. And because of the urgency of this PR (upload modal is broken), I don't have time to make it pretty :)